rules/windows/command_and_control_encrypted_channel_freesslcert.toml

```diff
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2020/11/04"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/04"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies unusual process connecting to domains using known free SSL Certificates. Adversaries may employ a known
```

Comment:
```suggestion
Identifies unusual processes connecting to domains using known free SSL Certificates. Adversaries may employ a known
encryption algorithm to conceal command and control traffic rather than relying on any inherent protections provided by
a communication protocol.
```